reports/sks444.md

```diff
@@ -0,0 +1,246 @@
+ - 27:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/59/diffs?commit_id=0581b4c81c981eca81541b7f312beabf97393537
+   - details: >
+     [Not Reviewed ] org/: Use igitt_django models to save igitt data
+ - 28:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/60/diffs?commit_id=25e750d6f4f26e3dc7a10fabd8897a23e27d4332
+   - details: >
+     [Not Reviewed] org/: Make an api to show active newcomers
+ - 29:
+   - repo: webservices.coala.io
+   - link: https://github.com/sks444/webservices.coala.io/commit/9d263725092f2ab18fde84fb46dececc95d2a76d
+   - details: >
+     Generate a fake api similar to coala's webservices
+
+### Newcomer Metrics and Gamification
+
+This project is about automating the recognition of activities performed by people
+who join the coala community, and encourage them to progress towards obtaining the
+role of "Developer" in the community.
+
+This project incorporated a gamification system which allows us to track the progress
+of newcomers activities and assign some points to each of the activities they do.
+The newcomers will get some score which will be the sum of all the points they have
+earned by performing several activities. The score they get will be a deciding factor
+for the current 'Level' of the newcomer and the activities they perform will be a
+deciding factor for 'Badges' they have earned.
+
+coala always tries to be a welcoming organization for newcomers, giving them clear
+pathways and as much direct assistance as possible. However, it is already known that
+newcomers face many barriers while attempting to contribute to open source for the
+first time, and additional barriers while attempting to contribute to projects run
+by the coala organization. On the other hand, gamification is widely used to engage
+and motivate people to accomplish tasks and improve their performance.
+
+Thus, the goal of this project is to use gamification to orient and motivate newcomers
+to overcome onboarding barriers to contribute and engage with coala and its community.
+
+#### Work Done
+
+##### Creating the APIs
+
+To build the gamification app, there was a need of constant data supply of all the
+activities newcomers do, i.e. Getting a pull request merge, Creating an issue and
+sending messages to the gitter channels. The work to achieve the same has been done
+in the [webservices](https://gitlab.com/coala/landing/) repo. And the output of all
+the script can be viewed at [webservices.coala.io](https://github.com/sks444/webservices.coala.io/commit/9d263725092f2ab18fde84fb46dececc95d2a76d) which is a workaround to get a fixed
+URL for all the APIs until there is the integration of continuous deployment in the webservices.
+
+##### Building the Gamification system
+
+The second and the important part of my work, from [importing the useful newcomers data](https://github.com/coala/community/commit/537602ecbe67263a1c3a877f88444ec6620190f4)
```

Comment:
avoid going over 80cpl as much as possible -- commits only need 8 characters, not the full sha.